app/javascript/utils/form/final_form/Field.js

```diff
@@ -24,14 +24,16 @@ export const BlockNote = ({ value }) => (
   <span className='checkbox-block_note no-margin padding-left--half'>{value}</span>
 )
 
-export const Input = ({ input, id, meta, type, maxLength, placeholder, ariaLabelledby, fieldName }) => (
+export const Input = ({ input, id, meta, type, maxLength, placeholder, ariaLabelledby, fieldName, disabled }) => (
```

Comment:
nit: with this many items it might be good to split this out onto multiple lines